src/hooks/custom-hooks/useExtendedOrderDetails.ts

```diff
@@ -106,12 +106,16 @@ const useExtendedOrderDetails = ({
             };
         },
         get displayPaymentAmount() {
-            return removeTrailingZeros(
-                FormatUtils.formatMoney(
-                    Number(this.amount_display) * Number(roundOffDecimal(this.rate, setDecimalPlaces(this.rate, 6))),
-                    { currency: this.local_currency as TCurrency }
-                )
-            );
+            if (this.rate) {
```

Comment:
this.rate && this.amount_display ?

Reply:
it was complaining previously for this.rate not for amount_display, so i think this is fine